torch/_export/__init__.py

```diff
@@ -683,6 +773,47 @@ def _export(
     constraints = constraints or []
     kwargs = kwargs or {}
 
+    if not strict:
+        assert isinstance(f, torch.nn.Module)
+        assert len(preserve_module_call_signature) == 0
+        assert len(constraints) == 0, "dynamic shape NYI"
+        assert len(kwargs) == 0, "keyword arguments NYI"
+        out_spec = None
+        def _tuplify_outputs(aot_export):
+            def _aot_export_non_strict(mod, args, **kwargs):
+                class Wrapper(torch.nn.Module):
+                    def __init__(self, mod):
+                        super().__init__()
+                        self._export_root = mod
+
+                    def forward(self, *args, **kwargs):
+                        nonlocal out_spec
+                        flat_outs, out_spec = pytree.tree_flatten(self._export_root(*args, **kwargs))
+                        return tuple(flat_outs)
+
+                gm, sig = aot_export(Wrapper(mod), args, **kwargs)
+                strip_root = lambda x: x[len('_export_root.'):] if x.startswith('_export_root.') else x
+                sig.parameters = pytree.tree_map(strip_root, sig.parameters)
+                sig.buffers = pytree.tree_map(strip_root, sig.buffers)
+                sig.inputs_to_buffers = pytree.tree_map(strip_root, sig.inputs_to_buffers)
+                sig.inputs_to_parameters = pytree.tree_map(strip_root, sig.inputs_to_parameters)
+                sig.buffers_to_mutate = pytree.tree_map(strip_root, sig.buffers_to_mutate)
+                return gm, sig
+            return _aot_export_non_strict
+        ep_non_strict = _export_non_strict(f, args, {}, f.state_dict(), transform=_tuplify_outputs)
```

Comment:
Do we need to fakeify the args? Seems like _export_non_strict takes fake args? Or we can move the _convert_input_to_fake call to _export_non_strict?

Reply:
I think I will leave this part for later discussion when we're supporting dynamic shapes. Right now it shouldn't matter.